doozer/doozerlib/backend/rebaser.py

```diff
@@ -276,7 +281,7 @@ def _resolve_parents(self, metadata: ImageMetadata, dfp: DockerfileParser, image
         mapped_images: List[Tuple[str, bool]] = []
         for parent, original_parent in zip(parents, dfp.parent_images):
             if "member" in parent:
-                mapped_images.append(self._resolve_member_parent(parent["member"], original_parent, image_repo, uuid_tag))
+                mapped_images.append(await self._resolve_member_parent(parent["member"], original_parent, image_repo, uuid_tag))
```

Comment:
stylistic suggestion, to split these into 2 lines for readability

Reply:
Not against it but I'd rather not touch anything more than what's strictly needed for the fix